rskj-core/src/main/java/org/ethereum/core/BlockHeader.java

```diff
@@ -477,9 +485,19 @@ public String getShortHashForMergedMining() {
         return HashUtil.shortHash(getHashForMergedMining());
     }
 
+    public boolean isUMMBlock() {
```

Comment:
Same comments as above, shouldn't all blocks be UMM Blocks after the activation?